src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/tabType';
+
+type Props = {
+  selectedTabId: string,
+  tabs: Tab[],
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  selectedTabId,
+  tabs,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id)
+  || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={classNames({
+                'is-active': selectedTabId === tab.id,
```

Comment:
```suggestion
                'is-active': selectedTab.id === tab.id,
```
Let's try this to pass the test